types/gapi.youtube/index.d.ts

```diff
@@ -1894,7 +1902,7 @@ interface GoogleApiYouTubeVideoResource {
         /**
          * A map of thumbnail images associated with the video. For each object in the map, the key is the name of the thumbnail image, and the value is an object that contains other information about the thumbnail.
          */
-        thumbnails: GoogleApiYouTubeThumbnailItemResource[];
+        thumbnails: GoogleApiYouTubeThumbnailResource;
```

Comment:
The type here seems to be contradicted by the documentation, which would imply a `Map<string, X>` or `{ [key: string]: X }`.